[2024-01-28T16:28:07.690+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-16T00:00:00+00:00 [queued]>
[2024-01-28T16:28:07.739+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-16T00:00:00+00:00 [queued]>
[2024-01-28T16:28:07.750+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-28T16:28:07.789+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2023-03-16 00:00:00+00:00
[2024-01-28T16:28:07.811+0000] {standard_task_runner.py:60} INFO - Started process 4426 to run task
[2024-01-28T16:28:07.820+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2023-03-16T00:00:00+00:00', '--job-id', '1491', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmp5c0mf4gr']
[2024-01-28T16:28:07.832+0000] {standard_task_runner.py:88} INFO - Job 1491: Subtask get_stackoverflow_data
[2024-01-28T16:28:07.981+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-16T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-28T16:28:08.282+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2023-03-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-16T00:00:00+00:00'
[2024-01-28T16:28:10.614+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-28T16:28:10.628+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20230316T000000, start_date=20240128T162807, end_date=20240128T162810
[2024-01-28T16:28:10.694+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T16:28:10.726+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
